clients/hadoopfs/src/test/java/io/lakefs/ImmutablePagination.java

```diff
@@ -0,0 +1,56 @@
+package io.lakefs;
+
+import com.google.common.base.Optional;
```

Comment:
Can we use the standard [java.util.Optional](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html) instead?  (Maybe not if code is generated with a different Optional...)

Reply:
Sure, I thought the code prefered Google's package so we left it.

Reply:
This code is used in a single location - and already used 'if' to check the optional value. So I've updarted to check for null.